src/Core/src/Core.csproj

```diff
@@ -64,15 +64,18 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="nuget\**" Exclude="nuget\**\*.in.*" PackagePath="" Pack="true" />
+    <None Remove="nuget\**" />
+    <None Include="nuget\**" Exclude="nuget\**\*.in.*;nuget\**\net-windows\*" PackagePath="" Pack="true" />
+    <None Include="nuget\buildTransitive\net-windows\**" PackagePath="buildTransitive\net$(_MauiMinimumSupportedDotNetTfm)-windows$(MinimumWindowsTargetFrameworkVersion)" Pack="true" />
```

Comment:
Can you add more context on what this change is doing?

Reply:
Seems its pick the base targets and moving to correct tfm from for picking the correct windows tfm ?